hudi-common/src/main/java/org/apache/hudi/common/table/read/HoodieFileGroupReader.java

```diff
@@ -108,24 +98,25 @@ public HoodieFileGroupReader(HoodieReaderContext<T> readerContext, HoodieStorage
       TypedProperties props,
       long start, long length, boolean shouldUseRecordPosition) {
     this(readerContext, storage, tablePath, latestCommitTime, dataSchema, requestedSchema, internalSchemaOpt,
-        hoodieTableMetaClient, props, shouldUseRecordPosition, false, false, false,
-        InputSplit.fromFileSlice(fileSlice, start, length), Option.empty(), false);
+        hoodieTableMetaClient, props, ReaderParameters.builder().shouldUseRecordPosition(shouldUseRecordPosition).build(),
```

Comment:
Is this deprecated constructor still used?

Reply:
I think it is required by some external repo based on the PR that added it back in